### ISSUE TITLE
Simple question on syntax.

### ISSUE THREAD

**Issue by pschuegr** [vojtajina/node-di#10]
Hi, 

I've been using node-di quite a bit, so thanks for this.  I have a question about this line:

https://github.com/vojtajina/node-di/blob/master/lib/injector.js#L70

```
var inject = fn.$inject && fn.$inject || autoAnnotate(fn);
```

I wouldn't call myself a javascript master by any means, but I don't understand the difference between this and:

```
var inject = fn.$inject || autoAnnotate(fn);
```

Is this just a typo?

**Comment by vojtajina** [comment]
You are right. I guess I was drunk or stupid when I wrote it;-) @matjaz already fixed it c573641c5ab0ef32b6d621631876b3c0a010c815

**Comment by pschuegr** [comment]
Ok, thanks for the clarification!
